Source/SpacePort/SpacePort/StarWarsApi.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using RestSharp;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using System.Threading.Tasks;
+
+namespace SpacePort
+{
+    public class StarWarsApi
+    {
+        public string Path { get; } = "https://swapi.co/api/";
+        public int StartIndex { get; } = 1;
+        public int EndIndex { get; set; }
+
+        public RestClient Client = new RestClient("https://swapi.co/api/");
+
+        public async Task<Dictionary<string, int>> GetAsyncShipData()
+        {
+            EndIndex = 37;
+
+            Dictionary<string, int> tmpDict = new Dictionary<string, int>();
+
+            for (int i = StartIndex; i <= EndIndex; i++)
+            {
```

Comment:
I'm unsure if the property "EndIndex" is needed. You can do a foreach-loop instead of a for-loop when iterating over the API-result.